src/components/Login/Login.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from "react";
+import  {useDispatch}  from "react-redux";
+import { loginAction } from "../../store/reducers/userReducer";
+import { Avatar, Button, Grid, Link, TextField, Typography, Box, Container } from "@mui/material";
+import LockOutlinedIcon from "@mui/icons-material/LockOutlined";
+
+const Login: React.FunctionComponent = () => {
+  const dispatch = useDispatch();
+
+  return (
+    <Container component="main" maxWidth="xs">
+      <Box
+        sx={{
+          marginTop: 8,
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "center",
```

Comment:
perhaps it should be taken to `const styles`

Reply:
done